client/src/components/shared/Point.js

```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import { useKeyPress } from "../shared/_utils";
```

Comment:
Can you tell me how this works? 
You are importing a component from a folder with this syntax?
Does underscore mean something in `_utils`?

Reply:
If we need somewhere to store hooks, or any other special functions, we can store them in `_utils`, short for "utilities". Files in `_utils` can be used in any of the components in the parent folder.